__tests__/defaults.test.js

```diff
@@ -0,0 +1,13 @@
+import {defaults as d} from '../src/js/defaults';
+
+describe('Testing defaults', () => {
+  test('checkAny function works correctly', () => {
```

Comment:
this test description is not very helpful. 
I don't know yet what it should be doing. 

better would be to split these up as well. (one to two assertions per test is the sweet spot).
```js
describe('just a number', () => {
  it('returns the input value, ()
describe('just a string', () => {
  it('returns the input value', ()
...
describe('missing data', () => {
  it('returns default value for undefined', () =>
  it('returns default value for null)
...
```